### DISCUSSION TITLE
What happened to the `@floating-ui/components` project?

### DISCUSSION THREAD

**Discussion by segevfiner** [floating-ui/floating-ui#3008] [General]
As tippy.js is no longer being developed:  https://github.com/atomiks/tippyjs/discussions/991#discussioncomment-1748552. @atomiks mentioned that he was working on a replacement, something higher level than `@floating-ui/dom` which is essentially what popper was. Something that handles events, showing/hiding, focus management, and all the sorts of other complex logic that often accompanies a popover besides just positioning (Which is difficult enough to warrant its own library, hence floating-ui/popper).

But I see no mention of this project anywhere after the repo was moved to the floating-ui org, and there doesn't seem to be anything well maintained and stable to replace tippy.js at this point. Which is a shame, cause libraries i tried that are based on floating-ui were inconsistent and buggy, to the point of causing other stuff on the page to glitch due to the mere presence of a popover, an I would love to have a proper maintained library for this and not having to roll my own, which can easily spiral into the same pit of issues getting it working right...

**Comment by atomiks** [comment]
That project, while intended to have a vanilla JavaScript foundation originally without using any frameworks/libraries like React, is what is now the `@floating-ui/react` package — it's React-only, and offers a solution that's lower-level than "components", but it does handle all that stuff that was mentioned.

I don't have any intention to work on vanilla libraries like `tippy.js` again or make `@floating-ui/react` framework-agnostic at this point though, so you're right that there exists no first-party package like what `@floating-ui/components` was originally intended to be. Sorry!

**Reply by segevfiner** [reply]
Bummer. Guess I'm kinda screwed to stay with tippy.js for the time being... Cause other libraries I tried just had too many subtle and annoying bugs or bad behaviors... 

😢
